### ISSUE TITLE
An unexpected error occurred: "Path must be a string. Received null".

### ISSUE THREAD

**Issue by hbakhtiyor** [yarnpkg/yarn#4429]
**Do you want to request a *feature* or report a *bug*?**

bug

**What is the current behavior?**
after updating yarn to latest version(1.0.1) getting error during `yarn install`
```
yarn install v1.0.1
[1/4] Resolving packages...
[2/4] Fetching packages...
error An unexpected error occurred: "Path must be a string. Received null".
```

**If the current behavior is a bug, please provide the steps to reproduce.**

here's the [yarn-error.log](https://gist.github.com/hbakhtiyor/4599c23300998e8cd33c0692bb810ab1#file-yarn-error-log) file

**What is the expected behavior?**

**Please mention your node.js, yarn and operating system version.**
node.js: v8.5.0
yarn: 1.0.1
os: ubuntu 16.04 64bit

**Comment by BYK** [comment]
Hey there! Can you run it with `--verbose` and share the output with us please?

**Comment by hbakhtiyor** [comment]
```
yarn install v1.0.1
verbose 1.277 Checking for configuration file "/srv/users/acme/demo-app/.npmrc".
verbose 1.279 Checking for configuration file "/srv/users/acme/.npmrc".
verbose 1.281 Checking for configuration file "/usr/etc/npmrc".
verbose 1.282 Checking for configuration file "/srv/users/acme/demo-app/.npmrc".
verbose 1.283 Checking for configuration file "/srv/users/acme/demo-app/.npmrc".
verbose 1.284 Checking for configuration file "/srv/users/acme/.npmrc".
verbose 1.284 Checking for configuration file "/srv/users/.npmrc".
verbose 1.285 Checking for configuration file "/srv/.npmrc".
verbose 1.289 Checking for configuration file "/srv/users/acme/demo-app/.yarnrc".
verbose 1.29 Checking for configuration file "/srv/users/acme/.yarnrc".
verbose 1.291 Found configuration file "/srv/users/acme/.yarnrc".
verbose 1.292 Checking for configuration file "/usr/etc/yarnrc".
verbose 1.293 Checking for configuration file "/srv/users/acme/demo-app/.yarnrc".
verbose 1.294 Checking for configuration file "/srv/users/acme/demo-app/.yarnrc".
verbose 1.295 Checking for configuration file "/srv/users/acme/.yarnrc".
verbose 1.296 Found configuration file "/srv/users/acme/.yarnrc".
verbose 1.297 Checking for configuration file "/srv/users/.yarnrc".
verbose 1.298 Checking for configuration file "/srv/.yarnrc".
verbose 1.307 current time: 2017-09-14T02:51:30.745Z
[1/4] Resolving packages...
[2/4] Fetching packages...
verbose 4.647 Performing "GET" request to "https://registry.yarnpkg.com/@glimpse/glimpse-agent-node/-/glimpse-agent-node-0.22.15.tgz".
verbose 6.86 TypeError: Path must be a string. Received null
    at assertPath (path.js:28:11)
    at Object.isAbsolute (path.js:1238:5)
    at module.exports.exports.default (/usr/share/yarn/lib/cli.js:87585:16)
    at /usr/share/yarn/lib/cli.js:48898:63
    at Generator.next (<anonymous>)
    at step (/usr/share/yarn/lib/cli.js:92:30)
    at /usr/share/yarn/lib/cli.js:103:13
    at <anonymous>
error An unexpected error occurred: "Path must be a string. Received null".

```

**Comment by BYK** [comment]
I've reproduced this and a fix is on the way. Thanks!

**Comment by hbakhtiyor** [comment]
what was the issue?

**Comment by BYK** [comment]
@hbakhtiyor one of the packages in the chain was included with its version set to null. Check out the pull request of you're interested.

**Comment by ArnoldLaishram** [comment]
I am still facing the same issue in version 1.0.2

Heres the error i got.

yarn install v1.0.2
[1/4] Resolving packages...
[2/4] Fetching packages...
error An unexpected error occurred: "Path must be a string. Received null".
info If you think this is a bug, please open a bug report with the information provided in "/home/sudhakar/ql-server/yarn-error.log".
info Visit https://yarnpkg.com/en/docs/cli/install for documentation about this command.

node-version: v8.5.0
yarn-version: 1.0.2
os: ubuntu 16.04 64bit

**Comment by BYK** [comment]
The fix for this didn't make it into 1.0.2 so that's normal: https://github.com/yarnpkg/yarn/compare/v1.0.1...v1.0.2

I hope to release 1.1.0 next week which would have the fix. In the meantime, you can use the latest nightlies: https://yarnpkg.com/en/docs/nightly
